### PR TITLE
Use a custom theme for editor

### LINKED ISSUE #761: UI: Improving matching parentheses
The highlighting of matching parentheses is *very* subtle: 
bild
It needs a better highlight, like possibly changing the background colour of the character. 

Also -- I suggest adding something like the Emacs rainbow-delimiters mode which colours the matching parentheses in the same colour for improved matching:
bild
Image above also shows the clearer highlighting of the outermost parentheses.

### PR DESCRIPTION
**Pull request by jiayushe**
- Use background highlighting for bracket matching
image

- Fix #761

### CODE REVIEW

File: src/components/workspace/source.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
+ace.define("ace/theme/source",["require","exports","module","ace/lib/dom"], function(acequire, exports, module) {
```
Comment (geshuming) [inline src/components/workspace/source.js]: Can we add a comment indicating that this theme originated from the cobalt theme that we have been using. As well as comment on what changes we made?

Comment (jiayushe) [reply src/components/workspace/source.js]: Okay done.

**Comment by TobiasWrigstad** [comment]
I turned out I entered a dependency hell when upgrading node packages to be able to build cadet-frontend again. Will let you know ASAP.

**Comment by TobiasWrigstad** [comment]
I would like to suggest that -- since we are making changes to the theme -- we change the font of the code from Ubuntu Mono to Inconsolata, to nicely unify with the SICP look.

**Comment by jiayushe** [comment]
> I would like to suggest that -- since we are making changes to the theme -- we change the font of the code from Ubuntu Mono to Inconsolata, to nicely unify with the SICP look. (I mean to piggy back this change on this PR since we are theming.)

Hmm I checked the SICP website, the font that is being used seems to be `Droid Sans Mono`?

image

**Pull request merged**